packages/app/features/account/settings/personal-info.tsx

```diff
@@ -84,8 +88,14 @@ export const PersonalInfoScreen = () => {
   }
 
   useEffect(() => {
-    form.reset({ phone: user?.phone ?? '', xUsername: profile?.x_username ?? '' })
-  }, [profile?.x_username, user?.phone, form.reset])
+    const birthday = profile?.birthday ? parseSupabaseDateStringToDate(profile.birthday) : undefined
```

Comment:
I believe the month problem was that our db dates our UTC. 
```suggestion
    const birthday = profile?.birthday ? 
adjustUTCDateForTimezone(new Date(profile.birthday)) : undefined
    

```

Reply:
results are the same, applied